draft-packages/zen-off-canvas/KaizenDraft/ZenOffCanvas/components/Header.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 
-import { ColorScheme } from "@kaizen/component-library/draft/Kaizen/ZenNavigationBar"
+import { ColorScheme } from "@kaizen/draft-zen-navigation-bar/KaizenDraft/ZenNavigationBar/NavigationBar"
```

Comment:
I'm feeling pretty confident about this since there's not been any changes in Navigation bar
`diff -r packages/component-library/draft/Kaizen/ZenNavigationBar draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar`